### DISCUSSION TITLE
容器部署，查看日志发现是 MySQL 没有连上，执行sql报错，但是本地通过工具能正常连接数据库

### DISCUSSION THREAD

**Discussion by qianpeng-qp** [SonicCloudOrg/sonic-server#422] [Q&A] [answered]
![image](https://github.com/SonicCloudOrg/sonic-server/assets/58299552/7abd33c7-8ca6-4482-8ad3-7f5eca871dc1)

**Comment by ZhouYixun** [comment]
试下docker-compose down再up吧

**Comment by ZhouYixun** [comment, marked as answer]
![image](https://github.com/SonicCloudOrg/sonic-server/assets/56339314/3b8e6fdf-041d-4bc4-91b5-d3936efab593)

注意字符集
